### DISCUSSION TITLE
is "allowFileAccessFromFileURLs" still necessary on iOS?

### DISCUSSION THREAD

**Discussion by Dervol03** [ionic-team/capacitor#6124] [Q&A] [answered]
Hi everyone,

we recently had a pentest for our app and one of the issues raised during the process was that `allowFileAccessFromFileURLs` is `true` for the webview on iOS. However, it is not set for the android counterpart code. 
This raises the question whether it is still necessary for capacitor to operate correctly or just an historical oversight due to a necessity in the past?

**Comment by jcesarmobile** [comment, marked as answer]
Probably not since it was added 5 years ago before capacitor used a custom scheme, and using a custom scheme will block doing XHR/fetch calls to file:// urls anyway.

If you create an issue we can properly evaluate and probably remove it for next major release (capacitor 5) if we don't see any reason for keeping it.

**Reply by Dervol03** [reply]
Thanks for the fast reply, I opened an issue.

Do you guys have a rough estimate on version 5? Just that I can tell our security experts, if it's going to be rather half a year, a year or longer

**Reply by jcesarmobile** [reply]
No idea, sorry. I don't make that kind of decisions.

But not longer than a year, before August we should be targeting SDK 33 as that's going to be a Google Play requirement for new apps, so 8 months tops, hopefully much sooner.

**Reply by Dervol03** [reply]
well, all I asked for was a rough estimate and I'd say this is one :) So thanks!
